source/cloud-native-platform/guides/postgresql-singleserver-to-flexibleserver-migration-portal.html.md.erb

```diff
@@ -0,0 +1,29 @@
+---
+title: Postgresql single server to flexible server migration using Azure Portal
+last_reviewed_on: 2023-04-24
+review_in: 6 months
+weight: 10
+hide_in_navigation: true
+---
+
+# <%= current_page.data.title %>
+
+The Azure Portal can be used to perform an offline migration of a PostgreSQL Single Server instance to PostgreSQL Flexible Server.
+
+This simple method has lower chances of failure and no restrictions in terms of database objects it can handle.
+
+### Limitations
+
+- Will result in downtime for applications.
+- Cannot be used to migrate databases from single servers using General Purpose SKU to flexible servers using Burstable SKU.
```

Comment:
i don't think worth mentioning as I don't think we'll be using the burstable sku